yr2025_1003/member.ts

```diff
@@ -0,0 +1,49 @@
+import { MEMBER_ROLES, NAMES } from "./globals.ts";
+
+export class Member {
+    name: string;
+    role: keyof typeof MEMBER_ROLES;
+    proficiencies: Map<keyof typeof MEMBER_ROLES, number>;

```

Comment:
Not sure how I feel about this abstraction. I think it's a cool idea, but maybe it starts to get confusing